.eslintignore

```diff
@@ -0,0 +1 @@
+.jshintignore
```

Comment:
It's just that Github's view fails to show that it is not the file content, but in fact a symbolic link

This is a symbolic link, because eslint allows jshint syntax.